docs/advanced/writing-an-app-connector.md

```diff
@@ -6,109 +6,38 @@ All existing connectors target all instances of a specific app, and can be deplo
 
 ## Architecture
 
-The connectors are composed of a root project which implements the base TheGraph connector, and an underlying subgraph. Both components are specified below.
+The connectors are composed of a root project which implements the base TheGraph connector, and an underlying subgraph. If you don't have an app subgraph yet, please refer to our [documentation for creating app subgraphs](../app-subgraphs.md) and [TheGraph's documentation](https://thegraph.com/docs).
 
 ### The connector
 
-The "src" folder contains the TheGraph connector wrapper which simply uses "@aragon/connect-thegraph" to perform GraphQL queries, parse them, and present them in the form of entity objects consistent with the "@aragon/connect" API.
+The `src` folder contains the TheGraph connector wrapper which simply uses `@aragon/connect-thegraph` to perform GraphQL queries, parse them, and present them in the form of entity objects consistent with the `@aragon/connect` API.
 
-Use "index.ts" to pick which entities and objects are exposed to other packages.
+Use `index.ts` to pick which entities and objects are exposed to other packages.
 
-In "connect.ts", add functions that the entities of your connector will use. For example, the Vote entity will call its "castsForVote(...): Promise<Cast[]>" function. These functions all follow the same structure of (1) performing a query, (2) parsing the results of a query, and (3) wrapping and returning the results in the appropriate entity.
+In `connect.ts`, add functions that the entities of your connector will use. For example, the Vote entity will call its `castsForVote(...): Promise<Cast[]>` function. These functions all follow the same structure of (1) performing a query, (2) parsing the results of a query, and (3) wrapping and returning the results in the appropriate entity.
 
-Each of the steps described above is separated in the "entities", "parsers", and "queries" folders for clarity.
+Each of the steps described above is separated in the `entities`, `parsers`, and `queries` folders for clarity.
 
-Notice the file "queries/types.ts". This file is generated by a 3rd party tool, and provides Typescript types for GraphQL results. It is completely optional, but can be super handy when writing parsers.
+Notice the file `queries/types.ts`. This file is generated by a 3rd party tool, and provides Typescript types for GraphQL results. It is completely optional, but can be super handy when writing parsers.
```

Comment:
```suggestion
```

Reply:
We decided to remove the types because they didn't provide that much value, so I think we should remove this section from the documentation.